src/main/java/de/samply/share/client/model/centralsearch/UploadStats.java

```diff
@@ -1,15 +1,16 @@
 package de.samply.share.client.model.centralsearch;
 
-import org.simpleframework.xml.Element;
-import org.simpleframework.xml.Root;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
 
 /**
  * Represents the upload stats as received from central search.
  */
-@Root(name = "UploadStats")
+
+@JacksonXmlRootElement(localName = "UploadStats")
```

Comment:
Can be removed, because the class name is used by default.

Reply:
In the previous version, there were two annotations: @Root and @Element. With the new library, we have replaced them with @JacksonRootElement and @JacksonXmlProperty. @JacksonRootElement does not harm and leaving both annotations, in my humble opinion, shows better what we have done: replace both annotations of the previous library with the correspondent annotations of the new one, continuing with the same logic.

Reply:
But if you read the documentation of @JacksonXmlRootElement, it says:
> NOTE! Since 2.4 this annotation is usually not necessary

So I would remove it. The new lib just doesn't need an annotation if the name of the XML root element matches the class name.